tests/pyccel/scripts/classes/classes_3_C.py

```diff
@@ -0,0 +1,21 @@
+# pylint: disable=missing-class-docstring,  missing-function-docstring, missing-module-docstring
+#$ header class Point(public)
+#$ header method __init__(Point, double[:])
+#$ header method translate(Point, double[:])
```

Comment:
These are not scalars. Please stick to scalars until we have looked at #1443 , otherwise we are introducing further memory leaks

Reply:
Fixed